### DISCUSSION TITLE
Deleting directories

### DISCUSSION THREAD

**Discussion by mausch** [jarun/nnn#1763] [Q&A] [answered]
Hi, nnn newbie here.
I can't seem to get deleting directories right. I'll describe what happens:
1. I put the cursor over the dir I want to delete and press `x`
2. nnn asks "rm -rf hovered?"
3. I press enter
4. nnn goes back to terminal and I get asked "rm: descend into directory 'whatever' ?"
5. I enter 'y'
6. rm then asks me if I want to delete each file

Something's wrong here with all these confirmations... if I just enter `rm -rf whatever` on the terminal it deletes things without asking.

**Comment by azuline** [comment, marked as answer]
after step 2, if you input `y` instead of `enter`, it shouldn't create any additional prompts
